extensions/adobe/experience/journeyOrchestration/stepEvents/journeyStepEventDataFetchFieldsMixin.json

```diff
@@ -0,0 +1,96 @@
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Journey Orchestration Step Data Fetch Fields",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/journeyOrchestration/stepEvents/journeyStepEvent",
+    "https://ns.adobe.com/xdm/context/experienceevent"
+  ],
+  "description": "Journey Orchestration Data Fetch Field for a Step Event",
+  "definitions": {
+    "journeyStepEventDataFetchFields": {
+      "properties": {
+        "https://ns.adobe.com/experience/journeyOrchestration/stepEvents": {
+          "title": "Journey Step Event Data Fetch Field",
+          "type": "object",
+          "properties": {
+            "journey:fetchTotalTime": {
+              "title": "fetchTotalTime",
+              "description": "This Field describes the total amount of time spent in data fetch in millis during the step processing",
+              "type": "integer",
+              "minimum": -9007199254740991,
+              "maximum": 9007199254740991
+            },
+            "journey:fetchTypeInError": {
+              "title": "fetchTypeInError",
+              "description": "This Field defines if the fetch in error is on the platform or on a custom data source",
+              "type": "string"
+            },
+            "journey:fetchError": {
+              "title": "fetchError",
+              "description": "This Field describes the type of error that happens when the data fetch is processed. ",
+              "type": "string"
+            },
+            "journey:fetchErrorCode": {
+              "title": "fetchErrorCode",
+              "description": "This Field describes the Code for fetch error. Present if the error has a code, such as an HTTP one. For instance, if the actionExecError is http, the code 404 represents the HTTP 404 error ",
+              "type": "string"
+            },
+            "journey:fetchOrigError": {
+              "title": "fetchOrigError",
+              "description": "This Field tells if there are any timeout or origin error",
+              "type": "string"
+            },
+            "journey:fetchOrigErrorCode": {
+              "title": "fetchOrigErrorCode",
+              "description": "This Field ",
+              "type": "string"
+            },
+            "journey:fetchCount": {
+              "title": "fetchCount",
+              "description": "This Field gives the count , How many times the data is fetched, regardless of the type of source (AEP, custom)",
+              "type": "integer"
+            },
+            "journey:fetchAepTotalTime": {
+              "title": "fetchAepTotalTime",
+              "description": "The total amount of time taken to fetch the data from Data Platform in millis",
+              "type": "integer"
+            },
+            "journey:fetchAepCount": {
```

Comment:
Same comment about AEP in name

Reply:
Updated the description to have Adobe Experience Platform, but should we change the title as well, it will be a long name then

Reply:
You misunderstand.  I don't want *any* part of AEP/Adobe in the name...

Reply:
updated it to Platform, please let me know if it is fine.